PyInstaller/hooks/pre_safe_import_module/hook-win32com.py

```diff
@@ -20,12 +20,15 @@
 
 import os
 
+from PyInstaller.compat import is_win
 from PyInstaller.utils.hooks import logger, get_module_file_attribute
 
 
 def pre_safe_import_module(api):
-    win32com_dir = os.path.dirname(get_module_file_attribute('win32com'))
-    comext_dir = os.path.join(os.path.dirname(win32com_dir), 'win32comext')
-    logger.debug('win32com: extending __path__ with dir %r' % comext_dir)
-    # Append the __path__ where PyInstaller will look for 'win32com' modules.'
-    api.append_package_path(comext_dir)
+    # TODO find out better way how to specify platformm specific code in pre_import hooks.
+    if is_win:
```

Comment:
Why is this necessary? AFAIK pre_safe_import-hooks are only called if the module as found - which can not happen on e.g. Linux. Or did I overlook something?